src/seedu/addressbook/parser/Parser.java

```diff
@@ -54,7 +54,7 @@ public Parser() {}
      * @param userInput full user input string
      * @return the command based on the user input
      */
-    public Command parseCommand(String userInput) {
+    static public Command parseCommand(String userInput) {
```

Comment:
Ah, so you're the one who did use `static public` instead of `public static`, I believe a typo there. Be careful.